mesonbuild/backend/backends.py

```diff
@@ -1354,6 +1362,27 @@ def create_install_data_files(self):
         with open(install_data_file, 'wb') as ofile:
             pickle.dump(self.create_install_data(), ofile)
 
+    def guess_install_tag(self, fname, outdir = None):
```

Comment:
I'm trying to have this fully typed, so can we add full annotations here?

I think that should be:
```python
def guess_install_tag(self, fname: str, outdir: T.Optional[str] = None) -> T.Optional[str]:
```

Reply:
done.